### DISCUSSION TITLE
香橙派4/4B,vless tcp xtls splice， 应该可以跑满千兆

### DISCUSSION THREAD

**Discussion by shenlijun** [XTLS/Xray-core#88] [Performance]
**好一点的arm现在性能真的比入门x86强，体积小，价格也不贵**

Orange pi 的U比树莓派4B强一些，带aes
4G内存，16G EMMC，壳板新的400+

**体积小，放弱电箱无压力**，散热与稳定性待验证。

可能RK3399是当前arm跑满千兆的最低需求，只谈配置Orange Pi 4性价比可能要好于R4S，他俩默认都没有主动散热

Khadas VIM3上面的A311D 表现应该会更好，板子的做工看上去也要比Orange好一些，VIM3L上的S905D3或许也可以，但是贵，可选主动散热

全志H6或许也可以，本来手上有个H5，但是千兆G口坏了
![捕获](https://user-images.githubusercontent.com/8860591/102667378-29771e80-41c4-11eb-806a-a9973fba15a2.PNG)

**Comment by shenlijun** [comment]
有意R4S的的话不如试试Orange pi 4

下次可能会测一下VIM3，板子的做工好，有主动散热可选，稳定性应该会好。但是价格快赶上两个香橙派4了

**Comment by badO1a5A90** [comment]
过热时会不会自动降频

**Comment by shenlijun** [comment]
> 过热时会不会自动降频

会的，但是表现还可以，小核不动，大核会变，会一直持续下图的状态，温度不太变化，大核频率在1.8-1.6之间跳。如果有主动散热估计就没问题了
![捕获](https://user-images.githubusercontent.com/8860591/102668566-2df10680-41c7-11eb-8d7a-41069730bf2f.PNG)

**Comment by shenlijun** [comment]
看来Khadas VIM3上面的A311D 值得一测吖
但是一套下来快1000了，除了体积小和稳定性**可能**好就没什么优点了

**Comment by shenlijun** [comment]
现在市面上的开发板集成了太多的东西，不是纯做网络设备，价格会超出预期，或许将来有专门的DD也不一定

**Comment by UJX6N** [comment]
Khadas的东西好像不是软路由这个方向的，轻奢doge

**Comment by bbcbbk** [comment]
树莓派4b 2G内存版现在全新的裸板250块，跑个openwrt 1G内存都够了

**Reply by Lixuhuilll** [reply]
现在裸板900+。魔幻的市场

**Comment by shenlijun** [comment]
> 树莓派4b 2G内存版现在全新的裸板250块，跑个openwrt 1G内存都够了

树莓派跑千兆稍微差了一点点，就一点点
香橙派U强，内存大，还有EMMC，裸板300多，不用额外加TF卡

**Comment by shenlijun** [comment]
> Khadas的东西好像不是软路由这个方向的，轻奢doge

是啊，开发板集成的东西多，Khadas做工好一点，配件也不错，目前它有卡片板上最强的U

要是开发板把别的都砍了只剩U，EMMC，千兆口，就完美了，不过受众太小没啥利润，估计没人专门做

目前也就R4S有这个想法，不过没EMMC，价格还不便宜

**Comment by bbcbbk** [comment]
> > 树莓派4b 2G内存版现在全新的裸板250块，跑个openwrt 1G内存都够了
> 
> 树莓派跑千兆稍微差了一点点，就一点点
> 香橙派U强，内存大，还有EMMC，裸板300多，不用额外加TF卡

准备出了N1和树莓派3，入一个200的二手树莓派4b，能800多M够了，性价比第一；orangePi太贵了，没性价比，这价格都可以x86了

**Comment by shenlijun** [comment]
> > > 树莓派4b 2G内存版现在全新的裸板250块，跑个openwrt 1G内存都够了
> > 
> > 
> > 树莓派跑千兆稍微差了一点点，就一点点
> > 香橙派U强，内存大，还有EMMC，裸板300多，不用额外加TF卡
> 
> 准备出了N1和树莓派3，入一个200的二手树莓派4b，能800多M够了，性价比第一；orangePi太贵了，没性价比，这价格都可以x86了

确实，1.5倍得价格
